bitnami/mysql/values.schema.json

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
It seems like this file is rendered with a 4-space indentation. In this repo we're currently following 2-space indentation for JSON files (i.e. WordPress [`values.schema.json`](https://github.com/bitnami/charts/blob/master/bitnami/wordpress/values.schema.json#L26)). Could you fix this? I'd recommend formatting with the `jq` CLI tool.

Reply:
done